Writerside/topics/Server-Manager.md

```diff
@@ -1,49 +1,103 @@
+A server manager is the point of contact for game servers to obtain and report on the status of matches. A server
+manager may also, but is not required to, track game server capacity and add and remove capacity by utilizing one or
+more allocators. The server manager may call on or be called by a [match broker](Match-Broker.md) to perform its duties.
 
 The default Catena server manager is contained within the [default match broker](Match-Broker.md#catena-match-broker).
 
 ## Allocators
 
-Allocators utilize a simple interface that a server manager can use to allocate or deallocate game servers as necessary on a particular platform or provider. Examples may include a local process allocator, a docker container allocator, or a Multiplay allocator. Other components, such as an admin dashboard, may also be able to utilize allocators directly.
+Allocators utilize a simple interface that a server manager can use to allocate or deallocate game servers as necessary
+on a particular platform or provider. Examples may include a local process allocator, a docker container allocator, or a
+Multiplay allocator. Other components, such as an admin dashboard, may also be able to utilize allocators directly.
 
 ## Game server endpoints
 
-The server manager is the point of interaction for game servers. The game server SDK interacts with the server manager to run matches.
+The server manager is the point of interaction for game servers. The game server SDK interacts with the server manager
+to run matches.
 
-An example of the simple flow, running a single match, is included in a [mock game server script](https://github.com/CatenaTools/catena-tools-core/blob/main/mocks/gameserver.py). 
+An example of the simple flow, running a single match, is included in
+a [mock game server script](https://github.com/CatenaTools/catena-tools-core/blob/main/mocks/gameserver.py).
 
 ### `RequestMatch`
 
-When a game server is idle (or backfilling or running matches in parallel) and has room for a new match it can call `RequestMatch()`. This will request a match from a [match broker](Match-Broker.md), optionally meeting any requirements of the game server by including a <tooltip term="JMESPath">JMESPath</tooltip> expression. The result of this may be one match or that there are no matches are available meeting the requirements in which case the game server may want to periodically retry until it gets a match or just exit (user implementation specific). This first use of this call is also the first indication to the backend that a server is extant.
+When a game server is idle (or backfilling or running matches in parallel) and has room for a new match it can
```

Comment:
For this section and the other requests, lets pull in the openapi docs now that they are being generated. They are located in this directory: https://github.com/CatenaTools/documentation/tree/main/Writerside/apispec/openapi

Reply:
They can be under a collapsible section so they aren't the main draw.

Reply:
We can address this in a later standardization effort